docs/release_notes/v0.12.0-alpha.md

```diff
@@ -0,0 +1,74 @@
+# Spice v0.12-alpha (Apr 29, 2024)
+
+The v0.12-alpha release introduces Clickhouse and Apache Spark data connectors, adds support for limiting refresh periods for temporal datasets, and includes upgraded Spice Client SDKs compatible with Spice OSS.
+
+## Highlights
+
+- **Clickhouse data connector**: Use [Clickhouse](https://clickhouse.com) as a data source with the `clickhouse:` scheme.
+
+- **Apache Spark Connect data connector**: Use Apache [Spark Connect](https://spark.apache.org/spark-connect/) connections as a data source using the `spark:` scheme.
+
+- **Temporal data refresh period**: Set specific time windows for accelerated datasets refresh, ensuring only the most recent data retained.
+
+- **Optional ODBC data connector**: Use ODBC connections as a data source. This feature can be enabled by using the `odbc` cargo feature when building from the trunk.
```

Comment:
```suggestion
- **ODBC data connector**: Use ODBC connections as a data source using the `odbc:` scheme. The ODBC data connector is currently optional and not included in default builds. It can be conditionally compiled using the `odbc` cargo feature when building from the trunk.
```